code/modules/client/preferences/species_features/basic.dm

```diff
@@ -146,6 +170,23 @@
 /datum/preference/choiced/hairstyle/apply_to_human(mob/living/carbon/human/target, value)
 	target.set_hairstyle(value, update = FALSE)
 
+/datum/preference/choiced/hairstyle/create_default_value()
+	return "Bald"
```

Comment:
congratulations you have returned to baystation

Reply:
RETURN OF BALDIES

Reply:
The only time these default values will ever be used is if 
A. You have an invalid hairstyle on save load (likely, from a removed hairstyle)
B. `create_informed_default_value` picks an invalid hairstyle (one which is locked for example) 
Otherwise this value is unused